specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2021-07-01/examples/DeleteRegisteredIdentities.json

```diff
@@ -0,0 +1,12 @@
+{
+  "parameters": {
+    "subscriptionId": "77777777-d41f-4550-9f70-7708a3a2283b",
+    "resourceGroupName": "BCDRIbzRG",
+    "vaultName": "BCDRIbzVault",
+    "identityName": "dpmcontainer01",
+    "api-version": "2021-06-01"
```

Comment:
Please change API version here

Reply:
please change everywhere in example files

Reply:
Done